_episodes/03-merge_conflicts.md

```diff
@@ -189,6 +188,49 @@ git graph
 >
 > NB: If the two sets of changes you made *aren't* incompatible (e.g. you changed
 > separate parts of the file) you will not get a merge conflict!
+>
+> > ## Solution
+> >
+> > ```
+> > git switch -c experiment2
+> > # make changes to ingredients.md (say 1/2 lime)
+> > git stage ingredients.md
+> > git commit -m "Reduced lime"
+> > git switch main
+> > # make changes to the same line in ingredients.md (say 1/4 lime)
+> > git stage ingredients.md
+> > git commit -m "Reduce lime to balance coriander"
+> > git merge --no-edit experiment2
+> > ```
+> > {: .commands}
+> > This should give rise to a merge conflict:
+> > ```
+> > Auto-merging ingredients.md
+> > CONFLICT (content): Merge conflict in ingredients.md
+> > Automatic merge failed; fix conflicts and then commit the result.
+> > ```
+> > Resolve the conflicts. Then stage the file again, and view the graph:
+> > ```
+> > git stage ingredients.md
+> > git commit -m "Merged experiment2 into main"
```

Comment:
I forget what we've told students to do elsewhere (and we want to be consistent), but as you're essentially providing the default message, you could just do:

```suggestion
> > git commit
```